### DISCUSSION TITLE
Example of embedding custom script for populated inputs

### DISCUSSION THREAD

**Discussion by Arsikod** [react-hook-form/react-hook-form#5825] [General]
Just wanted to add an implementation example of how to embed custom scripts into populated input components of a form.
It uses useWatch, setValue, useFormContext and FormProvider.

Main takeaway is how to communicate between independent react components using RHF.

I believe you can add it as an example to V7 examples.

Feel free to add any suggestions!

Code: https://codesandbox.io/s/cold-rain-tqoul?file=/src/App.js

**Comment by bluebill1049** [comment]
Thanks for the example! I will leave it to the discussion section and users can refer to it as an example.
